src/cookbooks/looker/countries.md

```diff
@@ -88,4 +88,4 @@ Now, you should be able to see the Countries View in your Explore 🎉
 
 ![Country View in Explore Image]
 
-[Country View in Explore Image]: ../../assets/Looker_screenshots/countries_explore.png 
\ No newline at end of file
+[Country View in Explore Image]: ../../assets/Looker_screenshots/countries_explore.png 
```

Comment:
I can't actually tell the diff between these lines... 👀

Reply:
There was no newline at the end of the file, this PR added one (probably because frank's editor did that automatically on save.